deployment/scripts/install.sh

```diff
@@ -1,9 +1,19 @@
 #!/bin/bash
 set -ex
 
+function delete_logging() {
+  initialize_install_vars
+  echo "Attempting to delete supporting objects (may fail)"
+  # delete oauthclient created in template; we can't search for it by label. the rest is incidental.
+  oc process logging-support-template | oc delete -f - || :
+  oc delete all,templates,secrets --selector logging-infra
```

Comment:
I thought that when I had `,daemonset` in that list it didn't actually remove it.. i'll double check and make the appropriate changes.